### DISCUSSION TITLE
Is there a plan to move some SpiceDB HTTP API features from experimental?

### DISCUSSION THREAD

**Discussion by nebozdes** [authzed/spicedb#2138] [Q&A]
Hi everyone, 

Is there any plan to move endpoints from SpiceDB HTT API (https://www.postman.com/authzed/spicedb/folder/zaabw6o/experimental):

- //v1/experimental/registerrelationshipcounter
- /v1/experimental/countrelationships 

from experimental to main API?
If so - is there any planned release / PR to do it?

Thanks in advance,
Anton.

**Comment by vroldanbet** [comment]
Hi,

we don't have plans to moving those out of experimental because the implementation is, right now, not scalable. We have plans to reimplementing it in a way that causes overhead over the datastore.

**Reply by jzelinskie** [reply]
It'd be great if you (@nebozdes) could describe why you're interested in this API so we can understand the use case and prioritize it accordingly
